modelscope/trainers/cv/image_classifition_trainer.py

```diff
@@ -498,6 +498,8 @@ def evaluate(self,
             metric_options = self.cfg.evaluation.get('metric_options', {})
             if 'topk' in metric_options.keys():
                 metric_options['topk'] = tuple(metric_options['topk'])
+            elif len(CLASSES) < 6:
```

Comment:
add some comment for this patching logic

Reply:
Done.